jobs/config.json

```diff
@@ -892,6 +892,16 @@
   ]
 },
 
+"ci-kubernetes-e2e-prow-canary": {
+  "scenario": "kubernetes_prow",
```

Comment:
prow version does not need ssh keys stuff though, and it will make code look messy..

I guess I'd like to get something working first? Then refactor/combine some code later.